br/pkg/lightning/lightning.go

```diff
@@ -317,6 +321,29 @@ func (l *Lightning) run(taskCtx context.Context, taskCfg *config.Config, o *opti
 		failpoint.Return(nil)
 	})
 
+	failpoint.Inject("SetCertExpiredSoon", func(val failpoint.Value) {
+		rootKeyPath := val.(string)
+		rootCaPath := taskCfg.Security.CAPath
+		keyPath := taskCfg.Security.KeyPath
+		certPath := taskCfg.Security.CertPath
+		certBytes, err := os.ReadFile(certPath)
+		if err != nil {
+			panic(err)
+		}
+		if err := os.WriteFile(certPath+".old", certBytes, 0o600); err != nil {
+			panic(err)
+		}
+		if err := updateCertExpiry(rootKeyPath, rootCaPath, keyPath, certPath, time.Second*10); err != nil {
```

Comment:
maybe make it expire immediately, so all new connections just fail

Reply:
We need to let first connection success. Otherwise, Lightning is unable to start.